dev/NavigationView/NavigationView.cpp

```diff
@@ -2264,9 +2279,20 @@ void NavigationView::UpdateSingleSelectionFollowsFocusTemplateSetting()
 
 void NavigationView::OnSelectedItemPropertyChanged(winrt::DependencyPropertyChangedEventArgs const& args)
 {
+
     auto newItem = args.NewValue();
+    auto oldItem = args.OldValue();
 
-    ChangeSelection(args.OldValue(), newItem);
+    ChangeSelection(oldItem, newItem);
+
+    // Animate to be sure the selected item is visually higlighted!
+    // See #1395 for additional context
```

Comment:
>See #1395 for additional context [](start = 7, length = 32)

I don't think this referenced pull request is very helpful for describing why this is needed, could you extract the reasoning into a more descriptive comment?

Reply:
I have updated the comment to outline the reasoning discussed in #1395